src/navigation/Navigator.tsx

```diff
@@ -37,11 +37,15 @@ export const styles = StyleSheet.create({
     borderBottomWidth: 1
   },
   headerRightContainer: {
-    paddingRight: 15,
-    flexShrink: 1
+    paddingHorizontal: 15,
+    maxWidth: 60
```

Comment:
setting a maxWidth will unfortunately not do the trick thats why spent so much time on it.
The right container has to stay also flexible. Thats why i worked with proportion or you use flex-basis

Reply:
yes because this container gonna be used in different cases